### PR TITLE
THU/cypress_tests_improvements

### PR DESCRIPTION
**Pull request by csm-thu**
* add "relogin" command to preserve cookies & local storage instead of logging in again between tests
* add optional timeout in scenario parameters "edit" action to wait end of simulation run
* tests refactoring
* add test for "double file upload"

### CODE REVIEW

File: cypress/integration/userInfo/UserInfo-test.spec.js [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
+import { AppBar } from '../../commons/actions';
 
 describe('UserInfo features', () => {
-  it('can log in with Microsoft account account', () => {
-    cy.visit(PAGE_NAME.SCENARIO);
-    cy.login();
-  });
-
   it('can change the application language', () => {
+    cy.login();
```
Comment (jreynard-code) [inline cypress/integration/userInfo/UserInfo-test.spec.js:8]: Even if this is a single test here, could be interesting to put the code snippet 
```
before(() => {
    cy.login();
  });

  beforeEach(() => {
    cy.relogin();
  });
```
to uniformize tests?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM ;)
Minor changes to discuss if you want ;)

**Review by Nikeleos — COMMENTED** [review]
Some little comments., otherwise LGTM.

File: cypress/commons/actions/generic/AppBar.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { FILE_NAME, SUPPORT_URL } from '../../constants/generic/TestConstants';
+import { GENERIC_SELECTORS } from '../../constants/generic/IdConstants';
+
+// User profile menu elements
+function getUserProfileMenuButton() {
+  return cy.get(GENERIC_SELECTORS.userProfileMenu.button);
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/commons/actions/generic/AppBar.js:9]: Good point, I'll rename it to UserInfo to stay consistent with the UI component :+1:

**Pull request merged**